libexec/cli/image.create.exec

```diff
@@ -96,6 +102,19 @@ if [ -f "$SINGULARITY_IMAGE" ]; then
     fi
 fi
 
+# Get the inode option *before* creating the empty file
+# We want to exit before the 'dd' command
+INODE=""
+if [ -n "${SINGULARITY_INODESIZE:-}" ]; then
+    INODE="-i ${SINGULARITY_INODESIZE}"
+
+    if [ $(expr ${SINGULARITY_INODESIZE} % 1024) -ne 0 ]; then
```

Comment:
This check is unnecessary.  We are not specifying the size of inodes, but the number of inodes in relation to the size of the disk space.  

On my system there is a minimum and maximum ratio that can be specified.  (min 1024/max 67108864), but I don't know how system specific so I'm hesitant to say we should check that the value is between that min and max.  

If you specify a value outside of that range, it will issue a warning and then create a broken file system for you.  I think the warning is pretty good though, and I'm inclined to just let it go at that.  

```
$ singularity image.create --bytes-per-inode 256 --size 512 ~/blah2
Creating empty 512MiB image file: /home/vagrant/blah2
Formatting image with ext3 file system
mkfs.ext3: invalid inode ratio 256 (min 1024/max 67108864)
```